beacon-chain/state/stategen/migrate.go

```diff
@@ -31,9 +31,10 @@ func (s *State) MigrateToCold(ctx context.Context, fRoot [32]byte) error {
 		return nil
 	}
 
-	// Start at previous finalized slot, stop at current finalized slot.
+	// Start at previous finalized slot, stop at current finalized slot (it will be handled in the next migration).
 	// If the slot is on archived point, save the state of that slot to the DB.
-	for slot := oldFSlot; slot < fSlot; slot++ {
+slotLoop:
```

Comment:
What is with this addition of a goto statement? This is a logic change beyond simple renaming, no?

Reply:
+1, why are we using `goto` here ? Goto statements are avoided unless due to necessity.

Reply:
The idea was to use a named loop because `continue` is very far away from `for`. But I agree this is not necessary.